utils.py

```diff
@@ -100,37 +103,20 @@ def __getitem__(self, idx):
         return X, y
 
 
-def pong_state_processor(img, state_shape):
-    """Resize states to 64x64 with cropping suited for Pong.
+def state_processor(img, state_shape, crop_range):
+    """Resize states to `state_shape` with cropping of `crop_range`.
 
     Args:
         img (np.ndarray): Image to crop and resize.
         state_shape (tuple): Output shape.
+        state_shape (tuple): Range to crop.
 
     Return:
         np.ndarray: Cropped and reshaped to `state_shape` image.
     """
 
     # Crop image to 160x160x3, removes e.g. score bar
-    img = img[35:195, :, :]
+    img = eval("img"+crop_range)
```

Comment:
Komentarz jest nieaktualny. 

Od strony stylu, to zwyczajowo masz whitespace pomiędzy operatorami, czyli `"img" + crop_range`

Reply:
Wydawało mi się że przeszedłem autopepem przez ten plik, poprawię to.

Reply:
https://stackoverflow.com/questions/18171560/does-pep-8-require-whitespace-around-operators-in-function-arguments, jop powinny być odstępy moim i stack-a zdaniem też 😄 Wyjątek to `=` przy nazwanych argumentach.